userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -1788,16 +1788,23 @@ static int32_t init(scap_t* handle, scap_open_args *oargs)
 	//
 	// Find out how many devices we have to open, which equals to the number of CPUs
```

Comment:
`Find out how many devices we have to open, which equals to the number of online CPUs`
I would specify that the device are equals to the online CPUs

Reply:
I'd drop the comment entirely.